src/hooks/useStaking.js

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react'
+import { useMounted } from './useMounted'
+import { useOrgApps } from '../providers/OrgApps'
+import { useWallet } from '../providers/Wallet'
+
+export function useStaking() {
+  const mounted = useMounted()
+  const { account } = useWallet()
+  const { agreementApp } = useOrgApps()
+  const [stakeManagement, setStakeManagement] = useState(null)
+  const [loading, setLoading] = useState(true)
+
+  useEffect(() => {
+    async function getStakingInformation() {
+      const defaultValues = {
+        available: '0',
+        challenged: '0',
+        locked: '0',
+        tokenDecimals: 1,
+        total: '0',
+      }
+      try {
+        if (account) {
+          const disputableApps = await agreementApp.disputableApps()
```

Comment:
I made a change in another PR which broke the way `agreementApp` works (sorry!)

But if you want to get this working again for testing you can change `agreementApp` to `connectedAgreementApp`